src/plugins/linked.js

```diff
@@ -110,14 +118,15 @@ export default class LinkedPlugins implements IPluginManager {
     if (pjson.scripts && pjson.scripts.prepare) {
       if (!this.config.debug) this.out.action.start(`Running prepare script for ${p}`)
       await this.yarn.exec(['run', 'prepare'], {cwd: p})
-      fs.utimesSync(main, new Date(), new Date())
       this.out.action.stop()
     }
+    touch(main)
 
     return true
   }
 
   _save () {
+    this._data.updated_at = new Date()
```

Comment:
@dickeyxxx also, there is a race condition in the `refresh()` where something could be updated between when it is checked and when the updated_at is set causing it to miss the change on the next execution.  There would be a case where something might be checked twice, but would be better than missing a change.  To be explicit, I was thinking that this would be better.

```
  async refresh () : Promise<string[]> {
    let paths : string[] = []
    let updatedAt = new Date()
    for (let plugin of this._data.plugins) {
      // ...
    }
    if (paths.length > 0) {
      this._data.updated_at = updatedAt
      this._save()
    }
    // ...
  }
```